oaknational/Oak-Web-Application
# Lesq 867   new lesson mv query - LESQ-867

## Description

Music year: 1979

- Lesson overview uses lesson content and browse MV
- Lesson overview canonical used same query (old query, schema and code removed)
- Quiz code refactored to camelCase

There is some additional work to remove the quiz types from OWA and use the curriculum schema types for quizes, this is happening at the query level but not component. I have added a tech debt ticket on the Miro board to revisit this, seemed out of scope of this ticket.



## How to test

1. Go to https://deploy-preview-2460--oak-web-application.netlify.thenational.academy
Go to lesson overview pages, they should look the same
Go to canonical lesson overview pages, they should look the same.


## Checklist
